### PR TITLE
Feature/slim 2811 added cucumber test for power failure event log

### PR DESCRIPTION
**Pull request by jboon**

### CODE REVIEW

File: integration-tests/cucumber-tests-platform-smartmetering/src/test/resources/features/osgp-adapter-ws-smartmetering/management/FindEvents.feature [hunk 1]
```diff
@@ -17,6 +17,7 @@ Feature: SmartMetering Management - Find Events
     Then 0 standard events should be returned
       | DeviceIdentification | TEST1024000000001 |
 
+  @FindEvents
```
Comment (carlogeenen) [inline integration-tests/cucumber-tests-platform-smartmetering/src/test/resources/features/osgp-adapter-ws-smartmetering/management/FindEvents.feature:20]: Do not forget to remove this.

File: integration-tests/cucumber-tests-platform-smartmetering/src/test/resources/features/osgp-adapter-ws-smartmetering/management/FindEvents.feature [hunk 2]
```diff
@@ -48,3 +49,11 @@ Feature: SmartMetering Management - Find Events
       | EndDate              | 2015-09-05T00:00:00.000Z |
     Then 29 mbus events should be returned
       | DeviceIdentification | TEST1024000000001 |
+
+  Scenario: find power failure events from a device
+    When receiving a find power failure events request
+      | DeviceIdentification | TEST1024000000001        |
+      | BeginDate            | 1970-01-01T00:00:00.000Z |
```
Comment (carlogeenen) [inline integration-tests/cucumber-tests-platform-smartmetering/src/test/resources/features/osgp-adapter-ws-smartmetering/management/FindEvents.feature:56]: This is some date range. Can it be more specific?

Comment (jboon) [reply integration-tests/cucumber-tests-platform-smartmetering/src/test/resources/features/osgp-adapter-ws-smartmetering/management/FindEvents.feature:56]: The date range is always from the beginning of (java) time to the current date, today.

**Review by bvdzwet — CHANGES_REQUESTED** [review]

File: integration-tests/cucumber-tests-platform-smartmetering/src/test/java/org/opensmartgridplatform/cucumber/platform/smartmetering/glue/steps/ws/smartmetering/smartmeteringmanagement/FindPowerFailureEventsReads.java [hunk 3]
```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright 2016 Smart Society Services B.V.
+ * Copyright 2020 Smart Society Services B.V.
```
Comment (bvdzwet) [inline integration-tests/cucumber-tests-platform-smartmetering/src/test/java/org/opensmartgridplatform/cucumber/platform/smartmetering/glue/steps/ws/smartmetering/smartmeteringmanagement/FindPowerFailureEventsReads.java:2]: Copyright statement is incorrect. Earlier review comment was not completely dealt with.

**Pull request merged**